CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/Server.swift

```diff
@@ -0,0 +1,176 @@
+                    } else if method.hasPrefix("encryptionkey") {
+                        result = try self.encryptionkeyRequestHandler.handleRequest(method: method, args: args)
+                    } else if method.hasPrefix("blob") {
+                        result = try self.blobRequestHandler.handleRequest(method: method, args: args)
+                    } else if method.hasPrefix("datatype") {
+                        result = try self.datatypeRequestHandler.handleRequest(method: method, args: args)
+                    } else if method.hasPrefix("replicatorConfiguration") {
+                        result = try self.replicatorConfigurationRequestHandler.handleRequest(method: method, args: args)
+                    } else if method.hasPrefix("replicator") {
+                        result = try self.replicatorRequestHandler.handleRequest(method: method, args: args)
+                    } else if method.hasPrefix("expression") {
+                        result = try self.expressionRequestHandler.handleRequest(method: method, args: args)
+                    } else if method.hasPrefix("collation") {
+                        result = try self.collationRequestHandler.handleRequest(method: method, args: args)
+                    } else if method.hasPrefix("dataSource") {
+                        result = try self.dataSourceRequestHandler.handleRequest(method: method, args: args)
+                    } else if method.hasPrefix("function") {
+                        result = try self.functionRequestHandler.handleRequest(method: method, args: args)
+                    } else if method.hasPrefix("selectResult") {
+                        result = try self.selectResultRequestHandler.handleRequest(method: method, args: args)
+                    } else if method.hasPrefix("result") {
+                        result = try self.resultRequestHandler.handleRequest(method: method, args: args)
+                    } else if method.hasPrefix("basicAuthenticator") {
+                        result = try self.basicAuthenticatorRequestHandler.handleRequest(method: method, args: args)
+                    } else {
+                        throw ServerError.MethodNotImplemented(method)
+                    }
+                    if result != nil {
+                        body = ValueSerializer.serialize(value: result, memory: self.memory);
+                    }
+                }
+
+                if body != nil {
+                    return GCDWebServerDataResponse(text: body as! String)
+                } else {
+                    // Send 200 code and close
+                    return GCDWebServerDataResponse(text: "I-1")
+                }
+            } catch let error as NSError {
+                // Send 400 error code
+                let response = GCDWebServerDataResponse(text: error.localizedDescription)!
+                print("Error is : \(error.localizedDescription)")
```

Comment:
should we have this or remove it. client tests will display the message if test fails anyways

Reply:
Done